### PR TITLE
Remove orchestra and metered channel

### PR DESCRIPTION
**Pull request by sandreim**
Resolve https://github.com/paritytech/polkadot/issues/6043

The crates have been moved to their new home: https://github.com/paritytech/orchestra

### CODE REVIEW

**Comment by drahnr** [comment]
Should be merged once https://github.com/paritytech/orchestra/pull/2 is done, so polkadot could specify deps as `orchestra = { version = "0.0.2", git = "https://.", branch = "master" }`

File: node/metrics/Cargo.toml [hunk 1]
```diff
@@ -10,7 +10,7 @@ futures = "0.3.21"
 futures-timer = "3.0.2"
 gum = { package = "tracing-gum", path = "../gum" }
 
-metered = { package = "prioritized-metered-channel", path = "../metered-channel" , "version" = "0.2.0" }
+metered = { package = "prioritized-metered-channel", git = "https://github.com/paritytech/orchestra", rev = "4ed886b0e48a0d9b27a658abd72a0bb00fe5f076" }
```
Comment (ordian) [inline node/metrics/Cargo.toml:13]:
```suggestion
metered = { package = "prioritized-metered-channel", version = "0.0.2" }
```

File: node/overseer/Cargo.toml [hunk 2]
```diff
@@ -15,15 +15,16 @@ polkadot-node-primitives = { path = "../primitives" }
 polkadot-node-subsystem-types = { path = "../subsystem-types" }
 polkadot-node-metrics = { path = "../metrics" }
 polkadot-primitives = { path = "../../primitives" }
-orchestra = { path = "../orchestra" }
+orchestra = { git = "https://github.com/paritytech/orchestra", rev = "4ed886b0e48a0d9b27a658abd72a0bb00fe5f076" }
```
Comment (ordian) [inline node/overseer/Cargo.toml:18]:
```suggestion
orchestra = "0.0.2"
```

File: node/overseer/Cargo.toml [hunk 3]
```diff
@@ -15,15 +15,16 @@ polkadot-node-primitives = { path = "../primitives" }
...
 gum = { package = "tracing-gum", path = "../gum" }
 lru = "0.7"
 parity-util-mem = { version = "0.11.0", default-features = false }
 sp-core = { git = "https://github.com/paritytech/substrate", branch = "master" }
 async-trait = "0.1.57"
 
 [dev-dependencies]
-metered = { package = "prioritized-metered-channel", path = "../metered-channel" }
+metered = { package = "prioritized-metered-channel", git = "https://github.com/paritytech/orchestra", rev = "4ed886b0e48a0d9b27a658abd72a0bb00fe5f076" }
```
Comment (ordian) [inline node/overseer/Cargo.toml:26]:
```suggestion
metered = { package = "prioritized-metered-channel", version = "0.0.2" }
```

File: node/subsystem-types/Cargo.toml [hunk 4]
```diff
@@ -13,7 +13,7 @@ polkadot-node-primitives = { path = "../primitives" }
 polkadot-node-network-protocol = { path = "../network/protocol" }
 polkadot-statement-table = { path = "../../statement-table" }
 polkadot-node-jaeger = { path = "../jaeger" }
-orchestra = { path = "../orchestra" }
+orchestra = { git = "https://github.com/paritytech/orchestra", rev = "4ed886b0e48a0d9b27a658abd72a0bb00fe5f076" }
```
Comment (ordian) [inline node/subsystem-types/Cargo.toml:16]:
```suggestion
orchestra = "0.0.2"
```

File: node/subsystem-util/Cargo.toml [hunk 5]
```diff
@@ -26,7 +26,7 @@ polkadot-node-network-protocol = { path = "../network/protocol" }
 polkadot-primitives = { path = "../../primitives" }
 polkadot-node-primitives = { path = "../primitives" }
 polkadot-overseer = { path = "../overseer" }
-metered = { package = "prioritized-metered-channel", path = "../metered-channel" , "version" = "0.2.0" }
+metered = { package = "prioritized-metered-channel", git = "https://github.com/paritytech/orchestra", rev = "4ed886b0e48a0d9b27a658abd72a0bb00fe5f076" }
```
Comment (ordian) [inline node/subsystem-util/Cargo.toml:29]:
```suggestion
metered = { package = "prioritized-metered-channel", version = "0.0.2" }
```

**Review by ordian — APPROVED** [review]
Needs a `Cargo.lock` update as well.

**Review by vstakhov — APPROVED** [review]

**Pull request merged**